synapse/config/room.py

```diff
@@ -83,4 +96,40 @@ def generate_config_section(self, **kwargs: Any) -> str:
         # will also not affect rooms created by other servers.
         #
         #encryption_enabled_by_default_for_room_type: invite
+
+        # Override the default power levels for rooms created on this server, per
+        # room creation preset.
+        #
+        # The appropriate dictionary for the room preset will be applied on top
+        # of the existing power levels content.
+        #
+        # Useful if you know that your users need special permissions in rooms
+        # that they create (e.g. to send particular types of state events without
+        # needing an elevated power level).  This takes the same shape as the
+        # `power_level_content_override` parameter in the /createRoom API, but
+        # is applied before that parameter.
+        #
+        # Valid keys are some or all of `private_chat`, `trusted_private_chat`
+        # and `public_chat`. Inside each of those should be any of the
+        # properties allowed in `power_level_content_override` in the
+        # /createRoom API. If any property is missing, its default value will
+        # continue to be used. If any property is present, it will overwrite
+        # the existing default completely (so if the `events` property exists,
+        # the default event power levels will be ignored).
+        #
+        #default_power_level_content_override:
+        #    private_chat: {
+        #        "events": {
+        #            "com.example.myeventtype" : 0,
+        #            "m.room.avatar": 50,
+        #            "m.room.canonical_alias": 50,
+        #            "m.room.encryption": 100,
+        #            "m.room.history_visibility": 100,
+        #            "m.room.name": 50,
+        #            "m.room.power_levels": 100,
+        #            "m.room.server_acl": 100,
+        #            "m.room.tombstone": 100
+        #        },
+        #        "events_default": 1
+        #    }
```

Comment:
Sorry, quick thing I've only noticed just now. This isn't valid YAML. Using `{` / `}` syntax will only work in YAML if it not split across multiple lines.

The below should work though (and is more YAML-y):

```suggestion
        #default_power_level_content_override:
        #    private_chat:
        #        "events":
        #            "com.example.myeventtype" : 0
        #            "m.room.avatar": 50
        #            "m.room.canonical_alias": 50
        #            "m.room.encryption": 100
        #            "m.room.history_visibility": 100
        #            "m.room.name": 50
        #            "m.room.power_levels": 100
        #            "m.room.server_acl": 100
        #            "m.room.tombstone": 100
        #        "events_default": 1
```

Reply:
Fixed in 8a639df70. Wow, I thought I already hated YAML.